src/configs/dolt_branch.rs

```diff
@@ -0,0 +1,31 @@
+use crate::config::ModuleConfig;
+
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct DoltBranchConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub truncation_length: i64,
+    pub truncation_symbol: &'a str,
+    pub detect_file: Vec<&'a str>,
+    pub detect_folder: Vec<&'a str>,
+    pub disabled: bool,
+}
+
+impl<'a> Default for DoltBranchConfig<'a> {
+    fn default() -> Self {
+        DoltBranchConfig {
+            format: "on [$symbol$branch]($style)",
+            symbol: " ",
```

Comment:
There's some discussion around potentially switching to emojis only by default. Please preferably use an emoji symbol here, and please also update the presets per the [checklist](https://github.com/starship/starship/blob/master/CONTRIBUTING.md#new-module-checklist).